doc/api/globals.md

```diff
@@ -295,6 +295,58 @@ The `MessagePort` class. See [`MessagePort`][] for more details.
 
 This variable may appear to be global but is not. See [`module`][].
 
+## `navigator`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of the [Navigator API][]. Similar to [`window.navigator`][]
+in browsers.
+
+### `navigator.deviceMemory`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total amount of device memory in GiB, rounded to the nearest power of 2,
+between 0.25 and 8 GiB. Part of the [Device Memory API][].
+
+```js
+console.log(`This device has ${navigator.deviceMemory} GiB of RAM`);
+```
+
+### `navigator.hardwareConcurrency`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {integer}
+
+The number of logical processors.
+
+```js
+console.log(`This device has ${navigator.hardwareConcurrency} logical CPUs`);
+```
+
+### `navigator.platform`
```

Comment:
`Navigator.platform` seems to be deprecated 

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform

there is quite many deprecated things on the navigator object that i don't think we should include

Reply:
The [specification](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-platform) does not indicate any deprecation.

Reply:
Hmm 😕 
Confused... Seems like MDN and TypeScript have incorrectly marked it as deprecated?

https://stackoverflow.com/questions/38506517/deprecated-javascript-os-detection-techniques

Reply:
@jimmywarting MDN takes the term "deprecated" to mean "use of this API is discouraged" - not "this API is on the way to removal from the spec". Under MDN's meaning of the term this marking as deprecated is correct because most of the properties/methods return incorrect and/or inconsistent information. There are more reliable ways to get the information.

Of course it would be good if the reasoning was provided along with the tagging.

Reply:
Note that around https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-oscpu, the spec adds this big red warning about the entire set of `NavigatorID` members (including `platform`):

> Any information in this API that varies from user to user can be used to profile the user. In fact, if enough such information is available, a user can actually be uniquely identified. For this reason, user agent implementers are strongly urged to include as little information in this API as possible.

That — along with the fact that most of the members are useless because they’re required by the spec to either not ever return anything useful or to not reliably/deterministically return anything useful — is why we have it flagged as deprecated in MDN: As a “don’t use this” signal to developers, so we don’t end up causing developers to mistakenly try to use any of it.

For example, in the case of `platform`, the spec requirements are:

> Must return either the empty string or a string representing the platform on which the browser is executing, e.g. "<code>MacIntel</code>", "<code>Win32</code>", "<code>FreeBSD i386</code>", "<code>WebTV OS</code>".

So, given that browsers can choose to always return the empty string for it, that means developers cannot and should not write code which assumes it returns "<code>MacIntel</code>", "<code>Win32</code>", "<code>FreeBSD i386</code>", "<code>WebTV OS</code>" or anything else actually useful. So it’s flagged as deprecated in MDN to help developers avoid making that mistake.

Reply:
Unless the spec have marked it as deprecated or removed it then i think you should use the term "Legacy" instead of "Deprecated" like NodeJS docs dose or something like that. 
Think "Deprecated" is a too strong word for what it actually means.

Say something in terms of: "Legacy discouraged fingerprinting, use feature detection instead"